app/assets/javascripts/solidus_paypal_braintree/braintree_errors.js.erb

```diff
@@ -0,0 +1,17 @@
+BraintreeError = {
```

Comment:
Sorry, just saw that this is a `js.erb` file instead of an html partial. Although not very obviously, but `js.erb` files are very problematic, because changes to the translations won't trigger this file to be re-compiled on next asset compilation. Please make this is html.erb file that gets rendered inside the payment form.

Reply:
Thank you for the feedback.
I changed this behavior and now it should work fine.